pkg/minishift/registration/utils.go

```diff
@@ -26,12 +26,17 @@ import (
 )
 
 type RegistrationParameters struct {
-	Username string
-	Password string
+	Username           string
+	Password           string
+	SkipRegistration   bool
+	SkipUnRegistration bool
 }
 
 // Register host VM
 func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
+	if param.SkipRegistration {
+		return nil
```

Comment:
What type of logging are you talking about here? glog? Something which appears when I turn on debug logs? Then yes, a general println to the console no.